corehq/apps/data_pipeline_audit/management/commands/find_sql_forms_not_in_es.py

```diff
@@ -64,12 +64,12 @@ def iter_form_ids_by_last_modified(start_datetime, end_datetime):
     return paginate_query_across_partitioned_databases(
         XFormInstance,
         (Q(last_modified__gt=start_datetime, last_modified__lt=end_datetime)
-         & Q(state=F('state').bitand(XFormInstance.DELETED)
-             + F('state').bitand(XFormInstance.DEPRECATED)
-             + F('state').bitand(XFormInstance.DUPLICATE)
-             + F('state').bitand(XFormInstance.ERROR)
-             + F('state').bitand(XFormInstance.SUBMISSION_ERROR_LOG)
-             + F('state'))),
+         & (Q(state=F('state').bitand(XFormInstance.DEPRECATED)
+              + F('state').bitand(XFormInstance.DUPLICATE)
+              + F('state').bitand(XFormInstance.ERROR)
+              + F('state').bitand(XFormInstance.SUBMISSION_ERROR_LOG)
+              + F('state'))
```

Comment:
I think this can be simplified to `Q(state__in=[XFormInstance.NORMAL, XFormInstance.ARCHIVED])`